docs/milestones.md

```diff
@@ -0,0 +1,8 @@
+#Project Milestones
```

Comment:
These project milestones need to be filled in. Links should be provided for the initial pitch video (if available) and the initial spec. See [the milestones section of the first assignment](https://github.com/reactivepixel/DWA/blob/master/ClassContent/Module-2-Pivot/Milestone-1.md#project-milestones) for further details.